tests/node_test.py

```diff
@@ -0,0 +1,148 @@
+import datetime
+import typing
+import unittest.mock
+
+from pytest import mark
+from pytest_localserver.http import WSGIServer
+from requests.exceptions import ConnectionError, Timeout
+from requests_mock import Mocker, mock
+from sqlalchemy.orm.scoping import scoped_session
+from typeguard import typechecked
+
+from nekoyume.models import Block, User
+from nekoyume.node import Node
+
+
+@typechecked
+def test_node(fx_server: WSGIServer, fx_session: scoped_session):
+    assert fx_server.url
+    assert Node.get(fx_server.url, session=fx_session)
+    assert Node.get(fx_server.url, session=fx_session).url == fx_server.url
+    assert Node.get(fx_server.url, session=fx_session).last_connected_at
+
+
+@typechecked
+def test_broadcast(fx_session: scoped_session, fx_user: User):
+    block = Block.create(fx_user, [])
+    url = 'http://test.neko'
+    now = datetime.datetime.utcnow()
+    node = Node(url=url, last_connected_at=now)
+    fx_session.add(node)
+    fx_session.flush()
+    with Mocker() as m:
+        m.post(url, text='success')
+        assert Node.broadcast(
+            '',
```

Comment:
Why is this empty?

Reply:
I think it is not necessary to set up endpoints in the test.